threejs/src/index.js

```diff
@@ -43,6 +45,26 @@ export default class ZestyBanner extends Mesh {
       }
     });
     this.onClick = this.onClick.bind(this);
+
+    setInterval(() => {
+      const camera = this.getCamera();
+      this.geometry.computeBoundingBox();
+      const bb = new Box3().setFromObject(this);
+      const isVisible = visibilityCheck(
+        bb.min.toArray(),
+        bb.max.toArray(),
+        camera.projectionMatrix.toArray(),
+        camera.matrixWorld.toArray(),
+      );
+      if (isVisible) {
+        loadBanner(adUnit, format, style).then(banner => {
+          this.material.map = banner.texture;
+          this.material.needsUpdate = true;
+          this.banner = banner;
+        });
+        console.log('Refreshed ' + adUnit);
```

Comment:
Do we still need this `console.log`?

Reply:
Another one that slipped through, removed